.evergreen.yml

```diff
@@ -79,8 +79,10 @@ functions:
           MCLI_PRIVATE_API_KEY: ${atlas_private_api_key}
           MCLI_PUBLIC_API_KEY: ${atlas_public_api_key}
           MCLI_SERVICE: cloud
+          E2E_CMD: gotestsum --junitfile unit-tests.xml --
+          E2E_TAGS: e2e,atlas
           <<: *go_options
-        command: make e2e-test-atlas
+        command: make e2e-test
   "cloud manager e2e":
```

Comment:
this can probably be just one function with different arguments but I'll change that later